### DISCUSSION TITLE
Unraid Alternative Setup

### DISCUSSION THREAD

**Discussion by clrksml** [tgorgdotcom/locast2plex#196] [Show and tell]
https://pastebin.com/raw/dGHg9tfC

 - Download the above rename it locast2plex.xml.
 - Create new folder in appdata called locast2plex.
 - Move file to new folder called locast2plex located in appdata.
 - Either use unraid console or ssh and use the following command 'mv /mnt/user/appdata/locast2plex/locast2plex.xml  /boot/config/plugins/dockerMan/templates -f'
 - Now go to docker tab and add new container locast2plex.
 - Congrats.
 
https://i.imgur.com/s0Lj3pP.png
https://i.imgur.com/wYQUomK.png
https://i.imgur.com/ae9IUGy.png
https://i.imgur.com/P3dIjDf.png

**Comment by tgorgdotcom** [comment]
Going to move this into discussions -> show and tell.  Thanks!

**Comment by wgstarks** [comment]
Is it still necessary to add variables to the docker settings for Plex IP, user/password and such?
